src/main/thrift/octopus.thrift

```diff
@@ -0,0 +1,62 @@
+namespace scala com.gu.flexibleoctopus.model.thrift
+
+/** date times are reprsented as i64 - epoch millis */
+typedef i64 DateTime
+
+/** 
+* A single story bundle that maps directly to a piece of content in Composer 
+*/
+struct StoryBundle {
+
+    1: required i64 id;
+
+    2: required string composerId;
+
+    3: required string pubCode;
+
+    4: required string sectionCode;
+
+    5: required Article article;
+
+}
+
+/**
+* Represents a single article of a story bundle as generated by inCopy, 
+* such as the headline, standfirst, or body text
+*/
+struct Article {
+
+    // id for the article
+    1: required i64 id;
+
+    // id for the book section
+    2: required i64 bookSectionId;
+
+    // indicates whether the article is for web and/or print pulication
+    3: required string forPublication;
```

Comment:
Would modelling this as a list provide more structure? IIRC this value in the payload is `w` for web, `p` for print and `b` for both?

Having it as a list would mean we have:

```
publicationPlatforms: [
  'web',
  'print'
]
```

Reply:
I don't _think_ you can have more than one value here, based on a demo @blishen did for us. I may well be wrong. Again, this is probably a candidate for being an enum.

Reply:
in the for publication thing - it breaks down like this:
W = for publication, for web, not for print
B = for publication, for web, for print
P = for publication, not for web, for print
w = not for publication, for web, not for print
b = not for publication, for web, for print
p = not for publication, not for web, for print